### PR TITLE
Hooked into `get_booking_products_args`.

### LINKED ISSUE #23: In edit booking, accommodation product is not selected in product field
screenshot

### PR DESCRIPTION
**Pull request by gedex**
This will make accommodation-booking included in the dropdown of
products.

Resolves #23 and #15.

### CODE REVIEW

**Comment by gedex** [comment]
@dwainm Please review.

File: includes/class-wc-accommodation-booking.php [hunk 1]
```diff
@@ -61,6 +62,32 @@ public function add_accommodation_to_booking_product_terms( $terms ) {
...
+	 *
+	 * @return array Query args
+	 */
+	public function add_accommodation_to_booking_products_args( $args ) {
+		foreach ( $args['tax_query'] as $index => $filter ) {
+			if ( 'product_type' !== $filter['taxonomy'] ) {
+				continue;
+			}
+
+			$terms = $args['tax_query'][ $index ]['terms'];
```
Comment (dwainm) [inline includes/class-wc-accommodation-booking.php]: @gedex is it possible that the data within $args['tax_query'][ $index ] may not exist? If so I'd add a check isset check at this point.

**Comment by dwainm** [comment]
@gedex left one note, besides this it looks good. Also fixes an issue on the add booking screen where the product was not selectable.

File: includes/class-wc-accommodation-booking.php (hunk 1 above)
Comment (gedex) [inline includes/class-wc-accommodation-booking.php]: @dwainm Good point. Though it's unlikely, I've added `isset` check in https://github.com/woothemes/woocommerce-accommodation-bookings/commit/791432688ff53fb1fb303210c5dd244c65c68027.

**Comment by dwainm** [comment]
Thanks, LG @gedex

**Pull request merged**